src/main/java/org/semver4j/internal/range/processor/RangesUtils.java

```diff
@@ -1,31 +1,36 @@
 package org.semver4j.internal.range.processor;
 
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
 import static java.lang.Integer.parseInt;
 
 /**
  * Set of methods which helps ranges handling.
  */
 final class RangesUtils {
+    @NotNull
     static final String EMPTY = "";
+    @NotNull
```

Comment:
That's not necessary in favor of field object assignment.

Reply:
Reasons 1 & 6